modele-social/règles/déclaration-revenu-indépendant.yaml

```diff
@@ -397,39 +397,65 @@ aide déclaration revenu indépendant 2020 . réduction covid . nombre de mois 
 
 
 aide déclaration revenu indépendant 2020 . revenu net fiscal:
+  applicable si: entreprise . imposition . IR
   titre: revenu net fiscal
   résumé: '[A]'
   description: Résultat avant déduction des charges sociales et exonérations fiscales 
   valeur: dirigeant . rémunération . totale
 
+
+
+aide déclaration revenu indépendant 2020 . rémunération nette dirigeant:
+  applicable si: entreprise . imposition . IS
+  résumé: '[X]'
+  description: | 
+    La rémunération nette du dirigeant déductible du résultat pour l'impôt sur le revenu (charges sociales non incluse).
```

Comment:
« déductible du résultat pour l'impôt sur le revenu » prête à confusion (j'imagine que c'est « déductible pour l'IS » d'un côté et à reporter pour le calcul de l'IR de l'autre)